File: EJERCICIO_2/EJERCICIOSJAVA/EJERCICIOJAVA/Person.java
package EJERCICIOJAVA;

public class Person {
    private String name;
    private int age;
    private float height;
    private float weight;

    public Person(String name, int age, float height, float weight) {
        this.name = name;   
        this.age = age;     
        this.height = height;   
        this.weight = weight; 
    }

    public String getName() {
        return this.name; 
    }

    public int getAge() {
        return this.age; 
    }

    public float getHeight() {
        return this.height; 
    }

    public float getWeight() {
        return this.weight; 
    }
}

File: EJERCICIO_2/EJERCICIOSPHP/person.php
<?php
class Person {
    private string $name;
    private int $age;
    private float $high;
    private float $weight;

    public function __construct($name, $age, $high, $weight) {
        $this->name = $name;   
        $this->age = $age;     
        $this->high = $high;   
        $this->weight = $weight; 
    }

    public function getName() {
        return $this->name; 
    }

    public function getAge() {
        return $this->age; 
    }

    public function getHigh() {
        return $this->high; 
    }

    public function getWeight() {
        return $this->weight; 
    }
}
?>